brambling/tests/functional/test_invites.py

```diff
@@ -19,7 +19,13 @@ def test_subject__event_editor(self):
         invite.send(Site('test.com', 'test.com'), content=invite.get_content())
         self.assertEqual(len(mail.outbox), 1)
         self.assertEqual(mail.outbox[0].subject, "{} has invited you to edit {}".format(invite.user.get_full_name(), event.name))
-
+    def test_subject__event_editor_apostrophe(self):
+	event= EventFactory(name="James's Test Event")
```

Comment:
There should also always be an empty line between the end of one method and the beginning of the next. It makes reading the code easier.